polkadot-parachains/src/service.rs

```diff
@@ -649,112 +649,59 @@ where
 			let telemetry2 = telemetry.clone();
```

Comment:
If we remove the second initialization, I doubt we need to have all these `clones()`. Can you remove them?

Reply:
Removed them, so we can proceed